### ISSUE TITLE
Unable to navigate between the fragments

### ISSUE THREAD

**Issue by criticalAY** [kiwix/kiwix-android#3246]
**Describe the bug**

Make a fresh install that has nothing to read (empty reader)
Click `open library`  this opens the library but when you click the reader section you won't be able to navigate back to the previous fragment

**Expected behavior**
able to navigate with ease

**Steps to reproduce the behavior:**
1. Make a fresh install
2. Click open library on reader section
3. click reader again

**Screenshots**

**Environment**
- Version of Kiwix Android : 3.6.0
- Device :  Emulator/Physical both
- OS version :  12

**Logs**

**Comment by kelson42** [comment]
Honestly not sure we have a problem here.

**Comment by criticalAY** [comment]
It makes the user experience not so good and we can't predict a user mindset and what they can do I am contributing to AnkiDroid and trust me user tend to do things that we can't predict :) 
Anyway this is one of the issues I hunted though we should let the user go back and forth as they are able to do so in the stable release which is on Playstore (I tested it out there and seems to work fine in that version like the way it should)
Secondly if we have a shortcut for `get content` and we click it to open the application then also we are not able to navigate to other fragments and this one is a major issue in my view

**Comment by criticalAY** [comment]
If you find this issue valid which in my opinion is then can I work on it?

**Comment by MohitMaliFtechiz** [comment]
> Honestly not sure we have a problem here. Why would the user want to come back here? What should be displayed by clicking on back?

@kelson42 , I have checked this , i think @criticalAY want to say clicking on reader button is not working. I have found here is a problem(other then if, we click on android back button reader is visible). we should fix this.



https://user-images.githubusercontent.com/34593983/222376525-437200e9-b971-469c-8133-38dfe5bd81c5.mp4

**Comment by kelson42** [comment]
How the solution could look like?

**Comment by kelson42** [comment]
@criticalAY Before making a PR, please explain how you plan to solve the problem. Once we agree on this, then you can start to code.

**Comment by criticalAY** [comment]
I will work around with the navigation graphs and will make sure that the buttons are unblocked so that the user can move around the fragments like they are able to do so in the stable release available in playstore

**Comment by MohitMaliFtechiz** [comment]
@kelson42, I have tested again this problem and found that this is a bug in both `Reader` and `LocalLibraryFragment`.
*  When we clicked on the `Download Books` button in the `Library screen` it will go to the `OnlineFragment` but if we click on the `Library` button in the bottom navigation then it does nothing like it is not working for the `reader` button.

I will place a fix for both issues soon.
